app/components/profile/ProfileComponent.jsx

```diff
@@ -209,6 +221,7 @@ class ProfileComponent extends React.Component {
                             }
                         </div>
                     </div>
+                    <div className="skillsInputStyle"><TextField type="text" name="name" value={this.state.skillsInput} floatingLabelText="Skills e.g.: excel, quickbooks,..." onChange={this.handleSkillsInput} /></div>
```

Comment:
name="name" shouldn't that be name="skills"

Reply:
yes updating